Finjector.Web/ClientApp/src/components/Folders/DeleteFolder.tsx

```diff
@@ -2,8 +2,8 @@ import React, { useState } from "react";
 import { Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
 import { useDeleteFolderMutation } from "../../queries/folderQueries";
 import { useNavigate } from "react-router-dom";
-import FinjectorButton from "../Shared/FinjectorButton";
-
+import { toast } from "react-toastify";
```

Comment:
perhaps we can find some way to hide and actual toast implementation from each client page?  For these kind of things I don't love exposing the actual library in case we need to change.  Perhaps a custom hook would be good, and that hook or method would be good, something that exposes a simple method or two for enqueuing errors and info messages